preloader/preload.go

```diff
@@ -66,6 +66,15 @@ func buildCloudConfig(script io.Reader, service io.Reader) (string, error) {
 		"systemctl daemon-reload",
 		"systemctl --no-block start customizer.service",
 	}
+
+	cloudConfig["bootcmd"] = []string{
+		"echo \"Resizing OEM partition file system...\"",
+		"umount /dev/sda8",
+		"e2fsck -fp /dev/sda8",
+		"resize2fs /dev/sda8",
+		"systemctl start usr-share-oem.mount",
```

Comment:
It might be safer to put --no-block here, in case cloud-init somehow end up having a dependency relationship with usr-share-oem.mount.